deliver/lib/deliver/runner.rb

```diff
@@ -72,10 +72,10 @@ def upload_metadata
     def prepare_app_icons(options = {})
       return unless options[:metadata_path]
 
-      default_app_icon_path = File.join(options[:metadata_path], "app_icon.png")
+      default_app_icon_path = Dir["#{options[:metadata_path]}/app_icon.{png,jpg}"].first
```

Comment:
Without using `File.join` there might be duplicate `/` between dir/file names. Any reason why we don't use File.join now?